packages/protocol/migrations/24_governance.ts

```diff
@@ -0,0 +1,79 @@
+/* tslint:disable:no-console */
+
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import {
+  deploymentForCoreContract,
+  getDeployedProxiedContract,
+  transferOwnershipOfProxy,
+  transferOwnershipOfProxyAndImplementation,
+} from '@celo/protocol/lib/web3-utils'
+import { config } from '@celo/protocol/migrationsConfig'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import { GovernanceInstance, GovernanceApproverMultiSigInstance } from 'types'
+
+const initializeArgs = async (): Promise<any[]> => {
+  const governanceApprovalMultiSig: GovernanceApproverMultiSigInstance = await getDeployedProxiedContract<
+    GovernanceApproverMultiSigInstance
+  >(CeloContractName.GovernanceApproverMultiSig, artifacts)
+
+  return [
+    config.registry.predeployedProxyAddress,
+    governanceApprovalMultiSig.address,
+    config.governance.concurrentProposals,
+    web3.utils.toWei(config.governance.minDeposit.toString(), 'ether'),
+    config.governance.queueExpiry,
+    config.governance.dequeueFrequency,
+    config.governance.approvalStageDuration,
+    config.governance.referendumStageDuration,
+    config.governance.executionStageDuration,
+    toFixed(config.governance.participationBaseline).toString(),
+    toFixed(config.governance.participationBaselineFloor).toString(),
+    toFixed(config.governance.participationBaselineUpdateFactor).toString(),
+    toFixed(config.governance.participationBaselineQuorumFactor).toString(),
+  ]
+}
+
+module.exports = deploymentForCoreContract<GovernanceInstance>(
+  web3,
+  artifacts,
+  CeloContractName.Governance,
+  initializeArgs,
+  async (governance: GovernanceInstance) => {
+    const proxyOwnedByGovernance = ['GoldToken']
+    await Promise.all(
+      proxyOwnedByGovernance.map((contractName) =>
+        transferOwnershipOfProxy(contractName, governance.address, artifacts)
+      )
+    )
+
+    //TODO @amyslawson do multisig implementations get added here?
```

Comment:
Great question!

They do not. The addresses that play the role of reserve spender/approver can be changed via governance, but the MultiSig should "own itself".

This can be accomplished by setting the proxy owner to the proxy address. The MultiSig signatories would then be able to upgrade the contract by submitting and confirming a MultiSig transaction via the proxy that calls `_setImplementation`.